snapshot-labs/sx-monorepo
# feat: Add Intercom messenger to UI

### To do:
- [x] Add `VITE_INTERCOM_APP_ID`  on vercel and fleek 

### Summary
Closes: #240 

Refer to the documentation here 
https://developers.intercom.com/installing-intercom/web/installation/

### How to test

- Go to any page on UI and you should see a widget in right side bottom, try reloading page and navigating to different pages and it should maintain its state
![image](https://github.com/snapshot-labs/sx-monorepo/assets/15967809/2edd74cd-8b72-4e1a-ad80-e199f44b7b43)
